scripts/dts/extract/heuristics.py

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (c) 2018 Bobby Noelte
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from extract.globals import *
+from extract.edts import *
+from extract.directive import DTDirective
```

Comment:
Do you really need this? Se following comment.

Reply:
Fixed